core/src/main/scala/org/http4s/rho/ServiceBuilder.scala

```diff
@@ -7,7 +7,7 @@ import shapeless.HList
 import scala.collection.immutable.VectorBuilder
 
 /** CompileService which accumulates routes and can build a `HttpService` */
-final class ServiceBuilder private(internalRoutes: VectorBuilder[RhoRoute.Tpe]) extends CompileService[RhoRoute.Tpe] {
+class ServiceBuilder (internalRoutes: VectorBuilder[RhoRoute.Tpe]) extends CompileService[RhoRoute.Tpe] {
```

Comment:
See the comments above about why we don't need to extend this. If you're going to change this file at all lets make the class private as well.